src/cursor/cur_backup.c

```diff
@@ -359,7 +359,7 @@ __wt_curbackup_open(WT_SESSION_IMPL *session, const char *uri, WT_CURSOR *other,
         /* Top level cursor code does not allow a URI and cursor. We don't need to check here. */
         WT_ASSERT(session, othercb == NULL);
         if (!F_ISSET(S2C(session), WT_CONN_INCR_BACKUP))
-            WT_RET_MSG(session, EINVAL, "Incremental backup is not configured");
+            WT_ERR_MSG(session, EINVAL, "Incremental backup is not configured");
```

Comment:
Is this a bug by itself? I would fix this in a separate ticket but it is up to you.

Reply:
Agreed moved both memory leaks into separate issue [WT-12659](https://jira.mongodb.org/browse/WT-12659)